oss-cloud-backend/getPullsHandler.js

```diff
@@ -0,0 +1,73 @@
+
+const access_token = process.env.GITHUB_ACCESS_TOKEN;
+const Octokit = require('@octokit/rest') // github api library
+const octokit = new Octokit({
+  auth: access_token,
+})
+octokit.hook.error('request', async (error, options) => {
+  throw error
+})
+
+// returns a single repo
+var getRepo = async (owner, repo) => {
+  return octokit.repos.get({
+    owner: owner,
+    repo: repo,
+  }).then(({data, headers, status}) => {
+    console.log("repo: ", data.name, " status", status)
+    return data;
+  })
+}
+
+module.exports.getForkedRepos = async(username) => {
+  return octokit.search.repos({
+    q: "user:" + username + "+fork:only"
+  }).then(({data, headers, status}) => {
+    return data.items;
+  })
+}
+
+
+// makes a call to github api for each repo in array, returning a more detailed representation of the repo
+module.exports.getRepoDetails = async (repos) => {
+  const repoPromises = repos.map(async (repo) => {

```

Comment:
'async' is not needed here

Reply:
Removing async caused a loading error. Function contains an await statement.

Reply:
but it doesn't?

Reply:
I was looking at the wrong function, removed async.